tools/loader/Loader.cpp

```diff
@@ -427,7 +449,19 @@ Loader::Loader(int argc, char **argv) {
     caffe2NetDescFilename_ = modelPathOpt[0];
     caffe2NetWeightFilename_ = modelPathOpt[1];
   }
+  M_.reset(new Module);
```

Comment:
Why do we need target specific code and all of this logic here? This totally complicates the loader.